hello.js

```diff
@@ -0,0 +1,9 @@
+// Function to compute the product of p1 and p2
+function printHelloName(name) {
+    for (let i = 0; i < 5; i++) {
+        const element = 5[i];
```

Comment:
what does `const element = 5[i];` do? Also, `return name;` is not needed, it will work without that. But looks great, overall